app/controllers/catalog_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
     @poh = PreservedObjectHandler.new(bare_druid, incoming_version, incoming_size, moab_storage_root)
     poh.create(checksums_validated)
     status_code =
-      if poh.results.contains_result_code?(:created_new_object)
+      if poh.results.contains_result_code?(:created_new_object) && !poh.results.contains_result_code?(:db_update_failed)
```

Comment:
bumped into these errors when getting existing tests to pass -- we do NOT want to send a 200 status code if we are able to create a new object but a later db update fails.

it may be that this combination is impossible, but I couldn't find a good way to update existing tests without adding this check.

Reply:
@jmartin-sul should I make a PR with this small change to catalog_controller, or is this just an oddity from how I originally implemented it?